Sources/Internal/Manager/CameraManager+MotionManager.swift

```diff
@@ -27,7 +27,7 @@ extension CameraManagerMotionManager {
 }
 private extension CameraManagerMotionManager {
     func handleAccelerometerUpdates(_ data: CMAccelerometerData?, _ error: Error?) {
-        guard let data, error == nil else { return }
```

Comment:
I believe with all of that, the `!` should be removed from:

``` swift
    private(set) weak var parent: CameraManager!
```

We should not pretend it can not be nil if it can.